src/ui/Base/Initialization.ts

```diff
@@ -283,18 +284,20 @@ export class Initialization {
     // this means that we can only execute the function after the promise has resolved
     if (resultOfSearchInterfaceInitialization.isLazyInit) {
       return resultOfSearchInterfaceInitialization.initResult.then(() => {
-        toExecuteOnceSearchInterfaceIsInitialized();
-        return {
-          elem: element
-        };
+        return toExecuteOnceSearchInterfaceIsInitialized().then(() => {
```

Comment:
It's possible to reduce one level of nesting here.

`return resultOfSearchInterfaceInitialization.initResult
.then(toExecuteOnceSearchInterfaceIsInitialized)
.then(() => {
return {
elem: element
}
}`